lib/utils/tagGenarator.js

```diff
@@ -0,0 +1,17 @@
+const TagModelManager = require('../../lib/modelManagers/tagModel');
+const { ArticleTags } = require('../../database/models');
+
+const generateTags = async (tag, id) => {
+  const tags = tag.split(',');
+  const keys = Array.from(Object.values(tags));
+  keys.map(async (EachTag) => {
+    await TagModelManager.findOrCreateTag(EachTag).then((data) => {
+      ArticleTags.create({
```

Comment:
Is there no `ArticleTags` model manager?

Reply:
there is none because article tags is a join table, I have used the add function provided by sequelize for join tables.